pkg/auth/converter.go

```diff
@@ -0,0 +1,36 @@
+package auth
+
+import (
+	"encoding/json"
+
+	"github.com/inspr/inspr/pkg/auth/models"
+	"github.com/inspr/inspr/pkg/ierrors"
+	"github.com/lestrrat-go/jwx/jwt"
+)
+
+// Desserialize converts a interface to a Payload model
+func Desserialize(tokenBytes []byte) (*models.Payload, error) {
+
+	token, err := jwt.Parse(tokenBytes)
+	if err != nil {
+		err = ierrors.NewError().InternalServer().Message("error: didn't return a token").Build()
```

Comment:
Might be better to specify the error in the message (err.Error), and I don't think "error:" is necessary